docs/en/offerings/spc/knowledge/connecting.md

```diff
@@ -7,16 +7,16 @@ lang: en
 
 # Connecting Your Studio With Your AWS VPC
 
-Clients can connect their studio networks with AWS using in one of many ways, described in detail in [Amazon Virtual Private Cloud Connectivity Options](https://docs.aws.amazon.com/whitepapers/latest/aws-vpc-connectivity-options/network-to-amazon-vpc-connectivity-options.html) whitepaper.
+You can connect your studio networks with AWS using one of many options, described in detail in this [Amazon Virtual Private Cloud Connectivity Options](https://docs.aws.amazon.com/whitepapers/latest/aws-vpc-connectivity-options/network-to-amazon-vpc-connectivity-options.html) whitepaper.
```

Comment:
let me know if you want me to show you how to extend the link so it covers "this Amazon.... whitepaper"